src/sst/core/testingframework/sst_unittest_support.py

```diff
@@ -75,6 +75,15 @@ def __str__(self) -> str:
 # Commandline Information Functions
 ################################################################################
 
+def testing_check_is_nightly() -> bool:
+    """If Nightly is in the BUILD_TAG it's very likely a Nightly build."""
+    build_tag = os.getenv("BUILD_TAG")
+    if build_tag and "Nightly" in build_tag:
```

Comment:
```suggestion
    if build_tag is not None and "Nightly" in build_tag:
```

Reply:
That's a little redundant don't you think?

Reply:
If `build_tag` was `None` then the `if` statement would fail...